src/test.rs

```diff
@@ -101,7 +103,18 @@ fn run_test(
             .unwrap();
         let extern_rs = std::fs::read_to_string(external_wasm_file_path).unwrap();
         wasm_lib_rs.write_all("\n\n".as_bytes()).unwrap();
-        wasm_lib_rs.write_all(extern_rs.as_bytes()).unwrap();
+        // we must replace the lib name if it's not the default
+        if let Some(custom_lib_name) = options.iter().find_map(|arg: &&str| {
+            arg.split_once("--lib-name=")
+                .and_then(|(_, lib_name)| Some(lib_name.replace('-', "_")))
```

Comment:
How do we have this configured? I didn't get this failure when I ran `cargo clippy` locally.

Reply:
You can find the args passed in for the Github action here: https://github.com/dcSpark/cddl-codegen/blob/master/.github/workflows/build.yml#L45